pkgs/development/haskell-modules/make-package-set.nix

```diff
@@ -281,46 +281,28 @@ in package-set { inherit pkgs stdenv callPackage; } self // {
       let
         selected = packages self;
 
-        packageInputs = map getBuildInputs selected;
-
-        name = if pkgs.lib.length selected == 1
-          then "ghc-shell-for-${(pkgs.lib.head selected).name}"
-          else "ghc-shell-for-packages";
+        pname = if pkgs.lib.length selected == 1
+          then (pkgs.lib.head selected).name
+          else "packages";
```

Comment:
Is there a reason for changing this from `ghc-shell-for-packages` to just `packages`?

I feel like `ghc-shell-for-packages` is easier to understand than just `packages` if all you see is the `/nix/store/HASH-ghc-shell-for-packages` path.

Reply:
I agree, and it's still there! The `ghc-shell-for-` is added by the `env` now.

Reply:
Ah, I see how this works.  Thanks for pointing this out.